jenkins/Jenkinsfile.j2

```diff
@@ -626,23 +626,30 @@ stage('Test') {
     SKIP_SLOW_TESTS = "${skip_slow_tests}"
   }
   parallel(
-  {% call(shard_index) m.sharded_test_step(
+  {% call(shard_index, num_shards) m.sharded_test_step(
     name="unittest: GPU",
     num_shards=2,
     node="GPU",
     ws="tvm/ut-python-gpu",
     platform="gpu",
   ) %}
+    {% if shard_index == 1 %}
     unpack_lib('gpu2', tvm_multilib)
     cpp_unittest(ci_gpu)
 
     unpack_lib('gpu', tvm_multilib)
     ci_setup(ci_gpu)
     cpp_unittest(ci_gpu)
+    {% else %}
+    unpack_lib('gpu', tvm_multilib)
+    ci_setup(ci_gpu)
+    {% endif %}
+    {% if shard_index == 2 or num_shards < 2 %}
```

Comment:
why shared_index == 2? first one with 'gpu' lib?

Reply:
one does c++ tests, the other does java